packages/showcases/css/stories/components/radio-button/examples/selected.html

```diff
@@ -0,0 +1,4 @@
+<div class="block">
+  <input type="radio"  id="my-radio-button" checked />
+  <label for="my-radio-button">Label</label>
```

Comment:
You need to change your 'for' and input 'id' because in showcase the three variants (disabled, selected and unselected) will be displayed together so the labels won't match correctly (there will be 3 similar label and id).

```suggestion
  <input type="radio"  id="my-selected-radio-button" checked />
  <label for="my-selected-radio-button">Label</label>
```